packages/itmat-job-executor/src/curation/JSONCurator.ts

```diff
@@ -122,16 +122,20 @@ export function processJSONHeader(header: string[]): {error?: string[], parsedHe
     const parsedHeader: Array<IFieldDescriptionObject | null> = Array(header.length);
     let colNum = 0;
     for (const each of header) {
-        if (!/^\d+@\d+.\d+(:[c|i|d|b|t])?$/.test(each)) {
-            error.push(`Object 1: '${each}' is not a valid header field descriptor.`);
-            parsedHeader[colNum] = null;
+        if (colNum === 0) {
+            parsedHeader[0] = null;
         } else {
-            const fieldId = parseInt(each.substring(0, each.indexOf('@')), 10);
-            const timepoint = parseInt(each.substring(each.indexOf('@') + 1, each.indexOf('.')), 10);
-            const measurement = parseInt(each.substring(each.indexOf('.') + 1, each.indexOf(':') === -1 ? each.length : each.indexOf(':')), 10);
-            const datatype: 'c' | 'i' | 'd' | 'b' | 't' = each.indexOf(':') === -1 ? 'c' : each.substring(each.indexOf(':') + 1, each.length) as ('c' | 'i' | 'd' | 'b');
-            parsedHeader[colNum] = { fieldId, timepoint, measurement, datatype };
-            fieldstrings.push(`${fieldId}.${timepoint}.${measurement}`);
+            if (!/^\d+@\d+.\d+(:[c|i|d|b|t])?$/.test(each)) {
+                error.push(`Object 1: '${each}' is not a valid header field descriptor.`);
+                parsedHeader[colNum] = null;
+            } else {
+                const fieldId = parseInt(each.substring(0, each.indexOf('@')), 10);
+                const timepoint = parseInt(each.substring(each.indexOf('@') + 1, each.indexOf('.')), 10);
+                const measurement = parseInt(each.substring(each.indexOf('.') + 1, each.indexOf(':') === -1 ? each.length : each.indexOf(':')), 10);
+                const datatype: 'c' | 'i' | 'd' | 'b' | 't' = each.indexOf(':') === -1 ? 'c' : each.substring(each.indexOf(':') + 1, each.length) as ('c' | 'i' | 'd' | 'b');
```

Comment:
This type is already present in another file, it would be worth thinking about extracting them out.